docs/modis_SimCLR/README.md

```diff
@@ -0,0 +1,9 @@
+# SimCLR for MODIS data set (version 1.0)
```

Comment:
Please put this file and probably most of the
others under `ulmo/runs/SSL/First/`

Reply:
I have kept Notebooks in `docs/nb` but I'll let you
decide the best place for those.